discord/ext/commands/core.py

```diff
@@ -1902,7 +1902,7 @@ def is_nsfw():
     """
     def pred(ctx):
         ch = ctx.channel
-        if ctx.guild is None or (isinstance(ch, discord.TextChannel) and ch.is_nsfw()):
+        if ctx.guild is None or ((isinstance(ch, discord.TextChannel) or isinstance(ch, discord.Thread)) and ch.is_nsfw()):
```

Comment:
```suggestion
        if ctx.guild is None or (isinstance(ch, (discord.TextChannel, discord.Thread)) and ch.is_nsfw()):
```

`isinstance` can accept a tuple of types.

Reply:
Oh nice, that's pretty handy